src/components/CardProject.astro

```diff
@@ -0,0 +1,52 @@
+---
+interface Props {
+  img: {
+    classProp: string;
+    src: string;
+    alt: string;
+  };
+  title: string;
+  description: string;
+  skills: Array<string>;
+}
+
+const {
+  img: { classProp, src, alt },
+  title,
+  description,
+  skills,
+} = Astro.props;
+
+const skillToCSS = {
+  HTML: "text-orange-600",
+  CSS: "text-blue-600",
+  JavaScript: "text-yellow-600",
+};
+---
+
+<div class="p-6 flex flex-col items-center gap-x-6 shadow-md">
+  <div class="h-64">
+    <img class={classProp} src={src} alt={alt} />
+  </div>
+  <div class="relative h-full pb-8">
+    <h3
+      class="mt-6 text-lg font-semibold leading-7 tracking-tight text-gray-900"
+    >
+      {title}
+    </h3>
+    <p class="text-base font-semibold leading-6 text-slate-700 my-8">
+      {description}
+    </p>
+    <ul class="w-full flex justify-around absolute bottom-0">
+      {
+        skills.map((skill) => {
+          return skillToCSS[skill] ? (
+            <li class={skillToCSS[skill]}>{skill}</li>
+          ) : (
+            <li class="text-cyan-600">{skill}</li>
+          );
+        })
```

Comment:
Nice, using arrow fn we can omit return like this:
```jsx
        skills.map((skill) => ( skillToCSS[skill] ? (
            <li class={skillToCSS[skill]}>{skill}</li>
          ) : (
            <li class="text-cyan-600">{skill}</li>
          );
        ))
```

Reply:
Changed